test/fluture.test.js

```diff
@@ -6,11 +6,11 @@ const Readable = require('stream').Readable;
 const jsc = require('jsverify');
 const S = require('sanctuary');
 const FL = {
-  map: 'map',
-  bimap: 'bimap',
-  chain: 'chain',
-  ap: 'ap',
-  of: 'of'
+  map: 'fantasy-land/map',
+  bimap: 'fantasy-land/bimap',
+  chain: 'fantasy-land/chain',
+  ap: 'fantasy-land/ap',
+  of: 'fantasy-land/of'
 };
```

Comment:
FantasyLand comes with its own dependency tree which seems to me like unnecessary overhead. It's not much of a big deal, but if there's no practical difference, or even much of a code-reuse advantage, I consider it cleaner not to depend on it.